index.html

```diff
@@ -9,23 +9,26 @@
 </head>
 <body>
     <main>
-        <section class="container">
-            <h2 class="books__header--title">
-                Awesome Books
-            </h2>
+        <h2 class="books__header--title">
+            All awesome books
+        </h2>
```

Comment:
- [ ] Please use proper semantic HTML here by using the `<header>` tag for the document title, and the `<main>` tag for the rest of the document.